test/verify/check-services

```diff
@@ -90,7 +98,14 @@ ExecStart=/usr/local/bin/test-service %I
 
         unit_action_btn = '#service-valid .list-group-item:nth-child(1) .btn-group'
         file_action_btn = '#service-valid .list-group-item:nth-child(2) .btn-group'
-
+        # Selects Timer tab
+        b.click('#services-filter :nth-child(4)')
+        b.wait_visible('#services')
+        m.execute("systemctl start test.timer")
+        b.wait_in_text(svc_sel('test.timer'), "Today")
```

Comment:
I think this is our #1 source for race conditions in our tests: before you can wait for the text to be there, you should first wait for the element o be present.
In this case, add a line before: `b.wait_present(svc_sel('test.timer'))`